launch/launch/actions/execute_local.py

```diff
@@ -434,6 +445,11 @@ def printer(context, msg, timeout_substitutions):
         sigkill_timeout = [PythonExpression(
             ('float(', *self.__sigterm_timeout, ') + float(', *self.__sigkill_timeout, ')')
         )]
+        sigkill_subprocesses_timeout = [PythonExpression(
+            (
+                'float(', *self.__sigterm_timeout, ') + float(', *self.__sigkill_timeout,
+                ') + float(', *self.__sigkill_subprocesses_timeout, ')')
+        )]
```

Comment:
I'm doing the same as the other timers are doing, but I think that this substitution should be resolved when the `ExecuteLocal` action is executed, and not when the timer is.

Reply:
+1 for consistency. If you want, you can propose a refactor in a follow-up PR.